### ISSUE TITLE
alicloud_vpc_peer_connection: API endpoint is not correct

### ISSUE THREAD

**Issue by shankerwangmiao** [aliyun/terraform-provider-alicloud#5453]
Hi there,

The new `alicloud_vpc_peer_connection` resource introduced in PR #5432 cannot be correctly created. The default API endpoint in the terraform provider is `vpcpeer-openapi-share.aliyuncs.com` <https://github.com/aliyun/terraform-provider-alicloud/blob/9ddc1935f4df7fc91cef62ea09d3eff35beb624f/alicloud/connectivity/client.go#L4430> and cannot be connected. According to <https://next.api.aliyun.com/product/VpcPeer#endpoint>, it should be `vpcpeer.aliyuncs.com`.

This issue can currently be worked around by manually specifying the api endpoint of the vpcpeer product: 

``` terraform
provider "alicloud" {
  // omited
  endpoints {
    // XXX: vpcpeer api endpoint is not correct
    vpcpeer = "vpcpeer.aliyuncs.com"
  }
}
```

**Comment by xiaozhu36** [comment]
HI @shankerwangmiao This issue has been fixed by the version 1.187.0. Please have a check.

**Comment by shankerwangmiao** [comment]
Many thanks!
